Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -806,6 +850,12 @@ bool JitArm64::DoJit(u32 em_address, JitBlock* b, u32 nextPC)
   gpr.Start(js.gpa);
   fpr.Start(js.fpa);
 
+  if (js.noSpeculativeConstantsAddresses.find(js.blockStart) ==
+      js.noSpeculativeConstantsAddresses.end())
```

Comment:
C++20 has [`unordered_set::contains`](https://en.cppreference.com/w/cpp/container/unordered_set/contains) which might be easier to read (though it probably has no performance difference). (The same applies to the x64 JIT.)

Reply:
I think I'd prefer to do it like this for now and then move them over all at once. (This isn't the first place in JitArm64 where we're using this idiom.)

Reply:
That's fine. (I also noticed that the x64 JIT uses camelCase in its `IntializeSpeculativeConstants`, which is something that can be fixed elsewhere).